main.tsp

```diff
@@ -4,71 +4,76 @@ import "@typespec/versioning";
 using Http;
 using Versioning;
 
-@service(#{ title: "Pet Store", version: "1.0.0" })
+@service(#{ title: "Clustron API", version: "1.0.0" })
 @server("https://example.com", "Single server endpoint")
 @versioned(Versions)
-namespace PetStore;
+namespace Clustron;
 
 enum Versions {
   v1: "1.0.0",
 }
 
-@doc("Represent a Pet available in the PetStore")
-model Pet {
-  id: int32;
+@doc("User settings that are needed for access to the machine.")
+model UserSettings {
+  @doc("The computer username of the user.")
+  username: string;
 
-  @minLength(1)
-  name: string;
-
-  @minValue(0)
-  @maxValue(100)
-  age: int32;
-
-  kind: petType;
-}
-
-enum petType {
-  dog: "dog",
-  cat: "cat",
-  fish: "fish",
-  bird: "bird",
-  reptile: "reptile",
+  publicKey: string[];
```

Comment:
Storing public keys directly in `UserSettings` instead of using a separate model and API could lead to unnecessarily large requests. 

A typical 2048-bit RSA public key alone takes around 400–500 characters. If a user configures multiple keys, and `UserSettings` later gets used in other contexts, this could introduce additional and avoidable performance overhead.